app/views/assessment/components/Button.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
```

Comment:
is it possible to not have the assessment screen use a bespoke button component? could we be using the same button as the rest of the app?

Reply:
There was a discussion about this and we decided not to use that button yet since it was refactored and added to too often. That being said I would like to open a different PR to implement the button we use throughout the app.

Reply:
makes sense to me!